contributing.md

```diff
@@ -30,7 +30,7 @@ Want to contribute a new feature? WP-CLI is a mature project, and already chock-
 
 New to the WP-CLI codebase? Check out [issues labeled 'good-first-issue'](https://make.wordpress.org/cli/good-first-issues/) for a place to start. These issues are specially earmarked for new contributors.
 
-Once you've decided to commit the time to seeing your pull request through, please [follow our guidelines for creating a pull request](https://make.wordpress.org/cli/handbook/pull-requests/) to make sure it's a pleasant experience. See "[Setting up](https://make.wordpress.org/cli/handbook/pull-requests/#setting-up)" for details on making local modifications to WP-CLI. Keep in mind pull requests are [expected to have tests](https://make.wordpress.org/cli/handbook/pull-requests/#running-and-writing-tests) covering the scope of the change. Read through [our code review guidelines](https://make.wordpress.org/cli/handbook/code-review/) for a better understanding of how your pull request will be evaluated.
+Once you've decided to commit the time to see your pull request through, please [follow our guidelines for creating a pull request](https://make.wordpress.org/cli/handbook/pull-requests/) to make sure it's a pleasant experience. See "[Setting up](https://make.wordpress.org/cli/handbook/pull-requests/#setting-up)" for details on making local modifications to WP-CLI. Keep in mind pull requests are [expected to have tests](https://make.wordpress.org/cli/handbook/pull-requests/#running-and-writing-tests) covering the scope of the change. Read through [our code review guidelines](https://make.wordpress.org/cli/handbook/code-review/) for a better understanding of how your pull request will be evaluated.
```

Comment:
This is grammatically wrong, the original sentence was correct. Please revert this change.

Reply:
@schlessera Done